client/Packages/com.beamable.server/CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.19.18]
+### Changed
+- `async void` methods are not allowed in Microservices
```

Comment:
Let's repeat the "Use `async Promise` or `async Task` instead." advice here, too.

Reply:
done